src/lib/helpers.ts

```diff
@@ -23,6 +23,27 @@ export function getParameterByName(name: string, url?: string) {
   return decodeURIComponent(results[2].replace(/\+/g, ' '))
 }
 
+// #107 Only remove Gotrue parameters from url hash.
```

Comment:
Reading the code and the unit tests for it, I'm able to figure out what it does so I see the comment as being a little noisy.

But I see how also it is useful, as it points out to where the issue is from.

Reply:
Ok I removed it as there are no such comments elsewhere and it would become quickly noisy if we had a comment for each issue. Also git blame & git history provide these informations.